docs/shop_administration/product_categories.md

```diff
@@ -0,0 +1,92 @@
+---
+description: You can create a structure of tags that can be assigned to products to help categorize products, to assist users in searching. One product can belong to different categories. In 4.2 you create, manage and assign categories manually, no import is available. In future releases categories will be assigned automatically based on rules, export/import will be added, as well as rules for applying different sales conditions based on categories.
+---
+
+# Product categories
+
+With product categories you can organize products within the Product Catalog and create relationships between them. 
+One of the reasons for applying product categories is assisting users in searching for products.
+
+Each category can be assigned to multiple products, and each product can belong to multiple categories of different or similar character, for example:
+
+- Business Laptops
+- Windows OS Devices
+- Stock clearance
+
+## Product category tree
+
+Product categories are organized into a Category tree, which you can access from the **Product catalog** panel. The **Categories** screen is where you manually create a hierarchy of categories and manage category entries.
+
+![Product category tree](img/product_categories.png)
+
+## Enable product categories
+
+Before you can assign categories to products, make sure that the "Taxonomy Entry Assignment" field is added and set as required in all the [Product Types](../../content_model/#content-types) that define products from your Product Catalog which you want to categorize.
+
+Depending on your permissions, you might be able to add the field yourself:
+
+1. In the left panel, click **Product catalog** -> **Product Types** to open a list of available Product Types.
+2. To create a new Product Type, click the  **Create** button and skip to step 4.
+3. To edit an existing Product Type, click the **Edit** icon next to a name of the Product Type that you want to modify.
+4. In the **Editing Product Type** modal, drag and drop the "Taxonomy Entry Assignment" option from the **Field types** area to the **Field definitions** area.
+5. Expand the **New field type** that you just added and make the following settings:
+    * Set a name for the field, for example, "Product category"
+    * Select the **Required** custom_checkbox
+    * In the **Taxonomy** field, make sure that the "Product categories" option is selected
+6. Optionally, modify other Product Type settings as required.
+7. Click **Save** to save your changes.
```

Comment:
I think we've made it so that category field is always added automatically? @adamwojs

Reply:
Not that I have observed - if Adam confirms that yes, though, will it simply be a matter of saying "make sure that the field is present"?

Reply:
Providing this field is added automatically, we can skip steps 4 i 5. User just has to make sure the field is in the CT I guess.